src/app/components/LinkData/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`LinkData should correctly render metadata for links 1`] = `null`;
```

Comment:
This snapshot isnt snapshotting anything (it has problems with helmet), you might need to pick a different snapshot render depth

Reply:
Thanks! I think I've fixed it now 👍